registry/midas/calldata-MinterVault.json

```diff
@@ -0,0 +1,1230 @@
+        { "chainId": 999, "address": "0x89AB3b922e9aec6ae5D2220deBf343d137a098A0" },
+        { "chainId": 999, "address": "0x210376434c1591f05399e3F1EF3f98C6e63d370E" },
+        { "chainId": 8453, "address": "0x8978e327FE7C72Fa4eaF4649C23147E279ae1470" },
+        { "chainId": 8453, "address": "0x80b666D60293217661E7382737bb3E42348f7CE5" },
+        { "chainId": 8453, "address": "0x2fD18B0878967E19292E9a8BF38Bb1415F6ad653" },
+        { "chainId": 8453, "address": "0x263A7AcE5E77986b77DcA125859248fEED52383c" },
+        { "chainId": 8453, "address": "0x3aAc6fd73fA4e16Ec683BD4aaF5Ec89bb2C0EdC2" },
+        { "chainId": 8453, "address": "0xFecc6FDFF76fB2A2De42B787dC3D02B634a8b6D9" },
+        { "chainId": 8453, "address": "0xEa22F8C1624c17C1B58727235292684831A08d56" },
+        { "chainId": 9745, "address": "0x2b690Cab819A815732544aEb422474EfDc1B0615" },
+        { "chainId": 9745, "address": "0x4Ef9fF56162bD3Cb5073FB20DbD355C59084093f" },
+        { "chainId": 16661, "address": "0x72a93168AE79F269DeB2b1892F2AFd7eaa800271" },
+        { "chainId": 23294, "address": "0xD7Fe0e91C05CAfdd26dA4B176eEc2b883795BDcC" },
+        { "chainId": 42161, "address": "0x643f73A6a3Ffc5d6C6Be7c97Cc30422763CFb1d0" },
+        { "chainId": 42161, "address": "0x01bfF1379CE9f0877141a18670d4214dFAf630bE" },
+        { "chainId": 42161, "address": "0x2c851A37eF2d607F198DDB259309Dcd2B398E8f9" },
+        { "chainId": 42161, "address": "0xb285f7699206C88A0aEC8bb004e42793de8139e0" },
+        { "chainId": 42161, "address": "0x9815FffE5600cF71342579f0f3E0Dd8ccBd496D8" },
+        { "chainId": 42793, "address": "0xd65BFeB71271A4408ff335E59eCf6c5b21A33a70" },
+        { "chainId": 42793, "address": "0x75C32818ce59D913f9E2aeDEcd5697566Ff9aE4A" },
+        { "chainId": 42793, "address": "0x577617613C4FaC5A7561F8f3F2Cb128A560774Bc" },
+        { "chainId": 42793, "address": "0xBEf85e71EcD0517D0C1446751667891b04860753" },
+        { "chainId": 42793, "address": "0x80D8226fE5D67803AB2D9d56EE184f7C6C6692e5" },
+        { "chainId": 42793, "address": "0x70449bbB9e6bee4D1a53151940148F21026d50b9" },
+        { "chainId": 98866, "address": "0xb05F6aa8C2ea9aB8537cF09A9B765a21De249224" },
+        { "chainId": 98866, "address": "0x8F38A24d064B41c990a3f47439a7a7EE713BF8Dc" },
+        { "chainId": 98866, "address": "0x23dE49C9ECb8bAaF4aBDeD123FaFbb7D5b7a0eE2" },
+        { "chainId": 98866, "address": "0xe6F0C60Fca2bd97d633a3D9D49DBEFDF19636D8c" },
+        { "chainId": 98866, "address": "0xc4E4aCA6A81794562c46DA86c20dc652bA2Af25E" },
+        { "chainId": 98866, "address": "0x71DD2570a843B0D1c74FFAb23F348193F19F18B1" },
+        { "chainId": 747474, "address": "0x175A9b122bf22ac2b193a0A775D7370D5A75268E" },
+        { "chainId": 747474, "address": "0xcb7d9A25F7b9bdd0Eee77B1cEb2894D39deBca1C" },
+        { "chainId": 1440000, "address": "0x30FBc82A72CA674AA250cd6c27BCca1Fe602f1Bb" }
+      ],
+      "abi": [
```

Comment:
ABI section can be simplified to include only the functions being clear signed later in the "display" section